hbase-server/src/main/java/org/apache/hadoop/hbase/master/normalizer/SimpleRegionNormalizer.java

```diff
@@ -241,16 +147,4 @@ public int compare(NormalizationPlan plan1, NormalizationPlan plan2) {
     Collections.sort(plans, planComparator);
     return plans;
   }
-
-  private long getRegionSize(RegionInfo hri) {
-    ServerName sn = masterServices.getAssignmentManager().getRegionStates().
-      getRegionServerOfRegion(hri);
-    RegionMetrics regionLoad = masterServices.getServerManager().getLoad(sn).
-      getRegionMetrics().get(hri.getRegionName());
-    if (regionLoad == null) {
-      LOG.debug(hri.getRegionNameAsString() + " was not found in RegionsLoad");
-      return -1;
-    }
-    return (long) regionLoad.getStoreFileSize().get(Size.Unit.MEGABYTE);
-  }
```

Comment:
Any chance of a test to prove the new normalizer is doing as advertised? Thanks.

Reply:
Thanks for the review sir. Currently working on test with valid test cases of merge